pandas/tests/util/test_util.py

```diff
@@ -467,3 +467,11 @@ def test_set_locale(self):
 
         current_locale = locale.getlocale()
         assert current_locale == self.current_locale
+
+
+def test_make_signature():
+    # See GH 17608
+    # Case where the func does not have default kwargs
+    make_signature(validate_kwargs)
```

Comment:
what is this doing now and what does this fix?

Reply:
The case on line 476 raises a TypeError in py3.  See #17608.

Reply:
this works for me in py3 now, so its not a good enough example

Reply:
Before I dig into this, did you run `make_signature` with both cases in the test?  My last comment had a typo, should have said line 477.

Reply:
i ran make_signature(validate_kwargs)

Reply:
pls try the call in the next line `make_signature(deprecate_kwarg)`

Reply:
ok this is fine. but still put an expected result and test against that, otherwise lgtm.